actions/types.ts

```diff
@@ -265,6 +277,7 @@ export interface Campaign {
   latestDeckIds?: number[]; // deprecated
   investigatorData: InvestigatorData;
   chaosBag: ChaosBag;
+  chaosBagResults: ChaosBagResults;
```

Comment:
So the legacy campaigns on devices means we need to make this optional
`chaosBagResults?: ChaosBagResults;`

Pretty sure the code is already setup to handle it being optional (with an `|| DEFAULT` in the connect statement)